contracts/Peeranha.sol

```diff
@@ -667,19 +671,23 @@ contract Peeranha is IPeeranha, Initializable {
         return PostLib.getVotedUsers(posts, postId, replyId, commentId);
     }
 
-    /*function addUserRating(address userAddr, int32 rating, uint32 communityId) external {
+    function addUserRating(address userAddr, int32 rating, uint32 communityId) external {
         PostLib.addUserRating(userContext, userAddr, rating, communityId);
-    }*/
+    }
 
-    /*function setEnergy(address userAddr, uint16 energy) external {
+    function setEnergy(address userAddr, uint16 energy) external {
```

Comment:
The same here. Comment and add a comment explaining that this used only in testing environments.